sfm_pc/sql/areas.sql

```diff
@@ -1,8 +1,7 @@
 SELECT
   object_ref.uuid AS uuid,
   MAX(name.value) AS name,
-  MAX(division_id.value) AS division_id,
-  substring(MAX(division_id.value), position(':' IN MAX(division_id.value)) + 1, 2) AS country_iso,
+  substring(MAX(division_id.value), position(':' IN MAX(division_id.value)) + 1, 2) AS org_country_iso,
```

Comment:
I chose `org_country_iso` for brevity, but can make the change to `organization_country_iso` if that's preferred.

Reply:
I will change it to the more verbose name to unblock this PR from merging. We can always change it back!

Reply:
Yep, verbose is better. Remember that the Great Rename (https://github.com/security-force-monitor/sfm-cms/issues/640) is almost upon us!